.github/workflows/test01-new-new.yml

```diff
@@ -51,37 +51,30 @@ jobs:
       
       - name: Test the ERC20 value transfer
         run: cd tests/value-transfer/ERC20 && bash run_testcase.sh
-        
-      - name: Logging the kend.out of EN node when job was failed
-        if: failure()
-        run: |
-          cd servicechain-docker
-          docker-compose -f docker-compose-servicechain.yml --env-file $DOCKER_COMPOSE_ENV_FILE exec -T EN-0 cat /klaytn/log/kend.out
-
-      - name: Logging the kscnd.out of SCN node when job was failed
-        if: failure()
-        run: |
-          cd servicechain-docker
-          docker-compose -f docker-compose-servicechain.yml --env-file $DOCKER_COMPOSE_ENV_FILE exec -T SCN-0 cat /klaytn/log/kscnd.out
-
-      - name: Logging the kcnd.out of CN node when job was failed
+      
+      - name: Copy log files in the failed-logs directory in case of failure
         if: failure()
         run: |
-          cd servicechain-docker
-          docker-compose -f docker-compose-servicechain.yml --env-file $DOCKER_COMPOSE_ENV_FILE exec -T CN-0 cat /klaytn/log/kcnd.out
-
-      - name: Logging the kpnd.out of PN node when job was failed
+          mkdir failed-logs
+          docker logs CN-0 >& failed-logs/kcnd.out
+          docker logs PN-0 >& failed-logs/kpnd.out
+          docker logs EN-0 >& failed-logs/kend.out
+          docker logs SCN-0 >& failed-logs/kscnd.out
+      
+      - name: Upload log files in case of failure
         if: failure()
-        run: |
-          cd servicechain-docker
-          docker-compose -f docker-compose-servicechain.yml --env-file $DOCKER_COMPOSE_ENV_FILE exec -T PN-0 cat /klaytn/log/kpnd.out
-
+        uses: actions/upload-artifact@v3
+        with:
+          name: failed-logs
+          path: failed-logs
+          retention-days: 10
```

Comment:
The allowed maximum seems 90 days. Why did you keep it for only ten days? any other reason?

Reply:
When someone get a failure alert on slack, I want to check it out as soon as possible.
But it is good to change maximum retention period.